packages/edit-post/src/components/visual-editor/index.js

```diff
@@ -85,16 +85,21 @@ function MaybeIframe( {
 export default function VisualEditor( { styles } ) {
 	const {
 		deviceType,
+		isActive,
 		isTemplateMode,
 		wrapperBlockName,
 		wrapperUniqueId,
 	} = useSelect( ( select ) => {
 		const {
+			isFeatureActive,
 			isEditingTemplate,
 			__experimentalGetPreviewDeviceType,
 		} = select( editPostStore );
 		const { getCurrentPostId, getCurrentPostType } = select( editorStore );
 		const _isTemplateMode = isEditingTemplate();
+		const feature = _isTemplateMode
+			? 'welcomeGuideTemplate'
```

Comment:
If I'm not wrong the PostTitle component is not visible at all in the template mode (it's not rendered), meaning we should either change the element we focus on, or avoid focusing anything entirely.

Reply:
This will only happen for normal welcome guide now.